code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1373,12 +1373,19 @@
 
 /datum/reagent/healium/on_mob_metabolize(mob/living/L)
 	. = ..()
-	ADD_TRAIT(L, TRAIT_KNOCKEDOUT, type)
+	L.PermaSleeping()
 
 /datum/reagent/healium/on_mob_end_metabolize(mob/living/L)
-	REMOVE_TRAIT(L, TRAIT_KNOCKEDOUT, type)
+	L.SetSleeping(10)
 	return ..()
 
+/datum/reagent/healium/on_mob_life(mob/living/L)
+	. = ..()
+	L.adjustOxyLoss(-2, FALSE)
```

Comment:
```suggestion
	. = ..()
	if(L.stat < UNCONSCIOUS)
		return
	L.adjustOxyLoss(-2, FALSE)
```
This is my only balance concern. Don't think this should heal a conscious mob.
If you neutralize it via another gas then you need to eschew the benefits.

Reply:
I think that removing the healing from a conscious mob will remove the reason to combine it with other gases, for the oxy i could just remove it is not a big deal

Reply:
What do you think, @ExcessiveUseOfCobblestone ?
I don't have too much of a strong opinion here besides wanting to avoid regen sources stacking at potentially no cost.

Reply:
as cobby told me, i removed the oxyloss but i kept the other interactions, we'll see how it's going to play out